File: muebleria/src/main/java/pe/com/muebleria/model/base/Empleado.java
package pe.com.muebleria.model.base;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Empleado extends BaseResultado {
	
	private Integer codigo;
	
	private String nombre;
	
	private String apellido;
	
	private Integer puesto;
	
}

File: Mantenedores-main/app/src/main/java/com/example/mantenedores/entity/Empleado.kt
package com.example.mantenedores.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.io.Serializable

@Entity(tableName = "tb_empleados")
data class Empleado(
    @PrimaryKey @ColumnInfo(name = "codigo") val codigo: Int = 0,
    @ColumnInfo(name = "nombre") val nombre: String?,
    @ColumnInfo(name = "apellido") val apellidos: String?,
    @ColumnInfo(name = "puesto") val puesto: String?): Serializable{

    override fun toString(): String{
        return "Empleado(codigo=$codigo, nombre='$nombre', apellidos='$apellidos', puesto='$puesto')"
    }
}


